src/main/java/Equation.java

```diff
@@ -0,0 +1,58 @@
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet (name = "Equation", urlPatterns = {"/calc/equation"})
+
+public class Equation extends HttpServlet {
+    private static boolean isOperator(char character) {
+        return character == '+' || character == '-' || character == '*' || character == '/';
+    }
+    private static boolean isDelimiter(char character) {
+        return character == '(' || character == ')';
+    }
+    static boolean isVariable(char character) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4840007)